e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/ShellAO.java

```diff
@@ -85,6 +85,14 @@ public ShellAO execute(String command) {
         return this;
     }
 
+    public ShellAO inputText(String command) {
+        sleep(300, MILLISECONDS);
+        Utils.sendKeysWithSlashes(command);
+        sleep(300, MILLISECONDS);
+        actions().sendKeys(Keys.ESCAPE).perform();
+        return this;
+    }
+
```

Comment:
We already have the similar method `execute()`. Let's reuse that - create common method `execute(String command, CharSequence key)`, and inputText() will be execute(command, Keys.ESCAPE)

Reply:
Done